lib/elixir/test/elixir/file_test.exs

```diff
@@ -21,6 +21,328 @@ defmodule FileTest do
   use Elixir.FileCase
   import Regex, only: [escape: 1]
 
+  defmodule Rename do
+
+    # Following erlang's underlying implementation
+    #
+    # Renaming files
+    # :ok               -> rename_file_to_existing_file_default_behaviour
+    # {:error,:eisdir}  -> rename_file_to_existing_empty_dir
+    # {:error,:eisdir}  -> rename_file_to_existing_non_empty_dir
+    # :ok               -> rename_file_to_non_existing_location
+    # {:error,:eexist}  -> rename_file_to_existing_file
+    # :ok               -> rename_file_to_itself
+
+    # Renaming dirs
+    # {:error,:enotdir} -> rename_dir_to_existing_file
+    # :ok               -> rename_dir_to_non_existing_leaf_location
+    # {:error,??}       -> rename_dir_to_non_existing_parent_location
+    # :ok               -> rename_dir_to_itself
+    # :ok               -> rename_dir_to_existing_empty_dir_default_behaviour
+    # {:error,:eexist}  -> rename_dir_to_existing_empty_dir
+    # {:error, :einval} -> rename_parent_dir_to_existing_sub_dir
+    # {:error, :einval} -> rename_parent_dir_to_non_existing_sub_dir
+    # {:error,:eexist}  -> rename_dir_to_existing_non_empty_dir
+
+    # other tests
+    # {:error, :enoent} -> rename_unknown_source
+    # :ok               -> rename_preserves_mode
+    # throw exception   -> rename_dir_to_existing_file_with_bang!
+
+    use Elixir.FileCase
+
+    test :rename_file_to_existing_file_default_behaviour do
+      src   = tmp_fixture_path("file.txt")
+      dest  = tmp_path("tmp.file")
```

Comment:
Ahh, misunderstood.  I thought you were commenting on the newline (versus the space), but I clearly misread.